internal/services/v1/permissions_test.go

```diff
@@ -378,6 +379,7 @@ func TestLookupResources(t *testing.T) {
 				t.Run(fmt.Sprintf("%s::%s from %s:%s#%s", tc.objectType, tc.permission, tc.subject.Object.ObjectType, tc.subject.Object.ObjectId, tc.subject.OptionalRelation), func(t *testing.T) {
 					require := require.New(t)
 					client, stop, revision := newPermissionsServicer(require, delta, memdb.DisableGC, 0)
+					defer goleak.VerifyNone(t, goleak.IgnoreCurrent())
```

Comment:
it looks weird to have this before `defer stop()`, but that lets `IgnoreCurrent` capture the running server's connections goroutines and check for leaks after `stop()`